EIPS/eip-1812.md

```diff
@@ -242,7 +242,7 @@ A Name Claim
 struct Name {
 	address issuer;
 	address subject;
-	string name;
+	string value;
```

Comment:
this was intentional to show semantic meaning naming being possible

Reply:
@ppoliani I think this answer is for you